### PR TITLE
Tournaments

### PR DESCRIPTION
**Pull request by TPGPL**
- Fix single column tables for osu! tournaments
- Fix linking in entire Tournaments folder
- Fix `Countries that participated in osu! tournaments`
- Fix `osu! Tag Team Multiplayer Tournament`
- Fix flags

### CODE REVIEW

File: wiki/Tournaments/CWC/2014/en.md [hunk 1]
```diff
@@ -3,15 +3,15 @@ osu! Catch the Beat World Cup 2014
...
+The **osu! Catch the Beat World Cup 2014** (***CWC 2014***) is a country-based Catch the Beat tournament hosted by the [Tournament Management](https://osu.ppy.sh/g/26). It is the 3rd installment of the osu! Catch the Beat World Cup. The previous title holder is **![][flag_CL] Chile**.
 
 This competition has come to an end and resulted in the following podium:
 
 | Place                                                      | Country                                        |
 |------------------------------------------------------------|------------------------------------------------|
-| ![Gold Crown](/wiki/shared/GCrown.png "1st place")   | ![](/wiki/shared/flag/Kr.gif) **South Korea** |
-| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![](/wiki/shared/flag/De.gif) **Germany**     |
-| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | ![](/wiki/shared/flag/Se.gif) **Sweden**      |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place")  [Kr_gif]outh Korea** |
```
Comment (oralekin) [inline wiki/Tournaments/CWC/2014/en.md]: *derp*

File: wiki/Tournaments/CMT_4K/2017/en.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ All prizes are sponsored by **汉斯**, **音游网** and **CrazyBoard**.
 
 ## Tournament stream
 
-bilibili: [aiyulu](http://live.bilibili.com/10183)(Primary), [Ookura Asahi](http://live.bilibili.com/272810)(Secondary), [Rasis](http://live.bilibili.com/5333)(Back up).   
+bilibili: [aiyulu](https://live.bilibili.com/10183)(Primary), [Ookura Asahi](https://live.bilibili.com/272810)(Secondary), [Rasis](https://live.bilibili.com/5333)(Back up).
```
Comment (kj415j45) [inline wiki/Tournaments/CMT_4K/2017/en.md]: bilibili live stream use http

File: wiki/Tournaments/MWC/2014/fr.md [hunk 3]
```diff
@@ -33,18 +33,18 @@ Organisation
...
-| Commentaires              | ![](/wiki/shared/flag/Us.gif) [Charles445](//osu.ppy.sh/u/85000) // ![](/wiki/shared/flag/Fr.gif) [Mr Color](//osu.ppy.sh/u/116078) // ![](/wiki/shared/flag/It.gif) [Roxas](//osu.ppy.sh/u/1041833) // ![](/wiki/shared/flag/Us.gif) [ztrot](//osu.ppy.sh/u/6347) // ![](/wiki/shared/flag/Ru.gif) [\[Shi-Ra\]](//osu.ppy.sh/u/2008489) |
-| Statistiques              | ![](/wiki/shared/flag/Pl.gif) [Marcin](//osu.ppy.sh/u/722665)                                                                                                                                                                                                                                                                                |
+| Direction                 | ![][flag_DE] [Loctav](https://osu.ppy.sh/u/71366) // ![][flag_DE] [p3n](https://osu.ppy.sh/u/123703) // ![][flag_ES] [Deif](https://osu.ppy.sh/u/318565)                                                                                                                                                  |
```
Comment (sr229) [inline wiki/Tournaments/MWC/2014/fr.md]: L36: Add Alt text to show which is supposed to be there.

File: wiki/Tournaments/MWC/2014/fr.md [hunk 4]
```diff
@@ -33,18 +33,18 @@ Organisation
...
+| Séléction des beatmaps    | ![][flag_RU] [\[Shi-Ra\]](https://osu.ppy.sh/u/2008489) // ![][flag_CL] [Agka](https://osu.ppy.sh/u/64811)                                                                                                                                                                                                            |
```
Comment (sr229) [inline wiki/Tournaments/MWC/2014/fr.md]: L37: Add Alt text to show which is supposed to be there.

File: wiki/Tournaments/MWC/2014/fr.md [hunk 5]
```diff
@@ -33,18 +33,18 @@ Organisation
...
+| Retransmission des matchs | ![][flag_AU] [peppy](https://osu.ppy.sh/u/2) // ![][flag_PL] [Marcin](https://osu.ppy.sh/u/722665) // ![][flag_FR] [shARPII](https://osu.ppy.sh/u/776257)                                                                                                                                                 |
```
Comment (sr229) [inline wiki/Tournaments/MWC/2014/fr.md]: L38: Add Alt text to show which is supposed to be there.

File: wiki/Tournaments/MWC/2014/fr.md [hunk 6]
```diff
@@ -33,18 +33,18 @@ Organisation
...
+| Commentaires              | ![][flag_US] [Charles445](https://osu.ppy.sh/u/85000) // ![][flag_FR] [Mr Color](https://osu.ppy.sh/u/116078) // ![][flag_IT] [Roxas](https://osu.ppy.sh/u/1041833) // ![][flag_US] [ztrot](https://osu.ppy.sh/u/6347) // ![][flag_RU] [\[Shi-Ra\]](https://osu.ppy.sh/u/2008489) |
```
Comment (sr229) [inline wiki/Tournaments/MWC/2014/fr.md]: L39: Add Alt text to show which is supposed to be there.

File: wiki/Tournaments/MWC/2014/fr.md [hunk 7]
```diff
@@ -33,18 +33,18 @@ Organisation
...
+| Statistiques              | ![][flag_PL] [Marcin](https://osu.ppy.sh/u/722665)                                                                                                                                                                                                                                                                                |
```
Comment (sr229) [inline wiki/Tournaments/MWC/2014/fr.md]: L40: Add Alt text to show which is supposed to be there.

File: wiki/Tournaments/OWC/2013/it.md [hunk 8]
```diff
@@ -24,15 +24,15 @@ Premi
 Organizzazione
 ---------------
 
-**Gestione torneo:** ![](/wiki/shared/flag/De.gif) [Loctav](//osu.ppy.sh/u/Loctav), ![](/wiki/shared/flag/Us.gif) [dkun](//osu.ppy.sh/u/dkun), ![](/wiki/shared/flag/De.gif) [p3n](//osu.ppy.sh/u/p3n)
+**Gestione torneo:** ![][flag_DE] [Loctav](https://osu.ppy.sh/u/Loctav), ![][flag_US] [dkun](https://osu.ppy.sh/u/dkun), ![][flag_DE] [p3n](https://osu.ppy.sh/u/p3n)
```
Comment (sr229) [inline wiki/Tournaments/OWC/2013/it.md]: L27: Add Alt text to show which is supposed to be there.

File: wiki/Tournaments/OWC/2013/it.md [hunk 9]
```diff
@@ -24,15 +24,15 @@ Premi
...
 
-**Riferimenti:** [Tournament Management usergroup](//osu.ppy.sh/g/26)
+**Riferimenti:** [Tournament Management usergroup](https://osu.ppy.sh/g/26)
 
-**Scelta delle beatmap:** ![](/wiki/shared/flag/Ar.gif) [Darksonic](//osu.ppy.sh/u/Darksonic), ![](/wiki/shared/flag/Ar.gif) [Wishy](//osu.ppy.sh/u/Wishy)
+**Scelta delle beatmap:** ![][flag_AR] [Darksonic](https://osu.ppy.sh/u/Darksonic), ![][flag_AR] [Wishy](https://osu.ppy.sh/u/Wishy)
```
Comment (sr229) [inline wiki/Tournaments/OWC/2013/it.md]: L31: Add Alt text to show which is supposed to be there.

File: wiki/Tournaments/OWC/2013/it.md [hunk 10]
```diff
@@ -24,15 +24,15 @@ Premi
...
 
-**Diretta trasmessa da:** ![](/wiki/shared/flag/Ca.gif) [Nyquill](//osu.ppy.sh/u/Nyquill), ![](/wiki/shared/flag/Au.gif) [kingking9](//osu.ppy.sh/u/kingking9), ![](/wiki/shared/flag/Us.gif) [Love](//osu.ppy.sh/u/Love), ![](/wiki/shared/flag/Ee.gif) [ShinseinaTenshi](//osu.ppy.sh/u/ShinseinaTenshi)
+**Diretta trasmessa da:** ![][flag_CA] [Nyquill](https://osu.ppy.sh/u/Nyquill), ![][flag_AU] [kingking9](https://osu.ppy.sh/u/kingking9), ![][flag_US] [Love](https://osu.ppy.sh/u/Love), ![][flag_EE] [ShinseinaTenshi](https://osu.ppy.sh/u/ShinseinaTenshi)
```
Comment (sr229) [inline wiki/Tournaments/OWC/2013/it.md]: L33: Add Alt text to show which is supposed to be there.

File: wiki/Tournaments/OWC/2013/it.md [hunk 11]
```diff
@@ -24,15 +24,15 @@ Premi
...
 
-**Commentatori:** ![](/wiki/shared/flag/Fr.gif) [Mr Color](//osu.ppy.sh/u/Mr_Color), ![](/wiki/shared/flag/Us.gif) [dkun](//osu.ppy.sh/u/dkun), ![](/wiki/shared/flag/Us.gif) [Agnes](//osu.ppy.sh/u/Agnes)
+**Commentatori:** ![][flag_FR] [Mr Color](https://osu.ppy.sh/u/Mr_Color), ![][flag_US] [dkun](https://osu.ppy.sh/u/dkun), ![][flag_US] [Agnes](https://osu.ppy.sh/u/Agnes)
```
Comment (sr229) [inline wiki/Tournaments/OWC/2013/it.md]: L35: Add Alt text to show which is supposed to be there.

**Pull request merged**